test/e2e/provision_libvirt.go

```diff
@@ -1,4 +1,5 @@
 //go:build libvirt
+// +build libvirt
```

Comment:
Do you really need both the formats ? The new format `go:build ...`  should be fine with >= go1.18

Reply:
Yeah, I tried both, unfortunately both not work, so added `//nolint:typecheck`

Reply:
Drooped the old one as added `//nolint:typecheck`

Reply:
Strange, why the error is happening. Let's look at it later on.